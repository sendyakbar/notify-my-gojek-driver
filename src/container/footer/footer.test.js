import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import Footer from './footer.jsx'

describe('<Footer> component', () => {
    const wrapper = shallow(<Footer />)
    test('renders correctly', () => {
        expect(wrapper).toHaveLength(1)
    })
    test('it has a footer', () => {
        const footerWrapper = wrapper.find('footer')
        expect(footerWrapper).toHaveLength(1)
    })
    test('it has a div', () => {
        const divWrapper = wrapper.find('div')
        expect(divWrapper).toHaveLength(1)
    })
    test('it renders correct and matched snapshot', () => {
        const tree = renderer.create(<Footer />).toJSON()
        expect(tree).toMatchSnapshot()
    })
})