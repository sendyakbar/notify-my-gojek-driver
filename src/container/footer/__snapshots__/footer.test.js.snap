// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer> component it renders correct and matched snapshot 1`] = `
<footer
  className="App-footer"
>
  <div
    className="foot-note"
  >
    <small>
      By sendyakbar Â©
      2019
    </small>
  </div>
</footer>
`;
